docs/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Current version only.
```

Comment:
```suggestion
```

Wondering if this is safe to remove given this is a website and not a library. It is probably worth having in our libraries though.